### ISSUE TITLE
Tile: Tile setting icon button has no text equivalent (1.1.1)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#952]
### Expected behavior
The settings/gear icon button has a text equivalent which can be read programmatically.  

### Actual behavior
There is no text equivalent as a label for the button.
